### PR TITLE
Brought longview-related functionality to API parity

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Brought longview-related functionality to API parity

## ✔️ How to Test

`pytest test`

Ticket: TPT-1889

### CODE REVIEW

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]

**Review by lgarber-akamai — APPROVED** [review]

File: linode_api4/groups/longview.py [hunk 1]
```diff
@@ -60,3 +64,37 @@ def subscriptions(self, *filters):
...
         """
         return self.client._get_and_filter(LongviewSubscription, *filters)
+
+    def longview_plan_update(self, longview_subscription):
+        """
+        Update your Longview plan to that of the given subcription ID.
+
+        :param longview_subscription: The subscription ID for a particular Longview plan.
+                                      A value of null corresponds to Longview Free.
+        :type longview_subscription: : str
```
Comment (zliang-akamai) [inline linode_api4/groups/longview.py:74]:
```suggestion
        :type longview_subscription: str
```

**Comment by zliang-akamai** [comment]
Are they two functions that do the same thing but in different location?

File: linode_api4/objects/longview.py [hunk 2]
```diff
@@ -24,3 +24,51 @@ class LongviewSubscription(Base):
...
+            )
+
+        params = {"longview_subscription": longview_subscription}
+
+        result = self._client.post(
+            LongviewPlan.api_endpoint, model=self, data=params
+        )
+        self.invalidate()
+
+        return LongviewPlan(self._client, result["id"], result)
```
[the remark this answers is not in the document]
Comment (zliang-akamai) [reply linode_api4/objects/longview.py:74]: It seems that you forget remove the original one, lol
@ezilber-akamai

**Comment by ezilber-akamai** [comment]
> Are they two functions that do the same thing but in different location?

No. I moved it from `object/longview.py` to `groups/longview.py` and forgot to remove it from `object/longview.py`. Good catch!

File: linode_api4/groups/longview.py [hunk 3]
```diff
@@ -60,3 +64,37 @@ def subscriptions(self, *filters):
...
         """
         return self.client._get_and_filter(LongviewSubscription, *filters)
+
+    def longview_plan_update(self, longview_subscription):
+        """
+        Update your Longview plan to that of the given subcription ID.
+
+        :param longview_subscription: The subscription ID for a particular Longview plan.
+                                      A value of null corresponds to Longview Free.
+        :type longview_subscription: str
```
Comment (yec-akamai) [inline linode_api4/groups/longview.py:74]: probably adding the `returns` and `rtype` as well?

**Review by yec-akamai — APPROVED** [review]

File: linode_api4/groups/longview.py (hunk 3 above)
Comment (ezilber-akamai) [reply linode_api4/groups/longview.py:74]: Fixed!

**Pull request merged**